public/app/views/profiles.html

```diff
@@ -4,27 +4,153 @@ <h2>Comfort Profiles</h2>
 	<div>
 		<p>Climate Mode
 		<select name="mode" ng-model="systems.config[0].mode[0]">
-			<option>off</option>
-			<option>fanonly</option>
-			<option>auto</option>
-			<option>heat</option>
-			<option>cool</option>
+			<option value="off">Off</option>
+			<option value="fanonly">Fan only</option>
+			<option value="auto">Auto</option>
+			<option value="heat">Heat</option>
+			<option value="cool">Cool</option>
 		</select>
 		</p>
 	</div>
+	<div ng-if="status.odu[0].type[0].includes('hp')">
+		<p>Heat Source
+		<select name="heatsource" ng-model="systems.config[0].heatsource[0]">
+			<option value="system">System in control</option>
+			<option value="odu only">Heat pump only</option>
+			<option value="idu only">Aux heat only</option>
+		</select>
+		</p>
+	</div>
+
+	<div class="panel panel-primary">
+		<div class="panel-heading"><strong>whole house</strong>
+			<br>Hold: <input type="checkbox" ng-model="systems.config[0].wholeHouse[0].hold[0]" ng-true-value="'on'" ng-false-value="'off'">
+			<span ng-if="systems.config[0].wholeHouse[0].hold[0] == 'on'">
+			<select name="holdact" ng-init="systems.config[0].wholeHouse[0].holdActivity[0] = 'home'" ng-model="systems.config[0].wholeHouse[0].holdActivity[0]">
+				<option>home</option>
+				<option>away</option>
+			</select>
+			Forever <input type="checkbox" ng-model="systems.config[0].wholeHouse[0].otmr[0]" ng-true-value="{}" ng-false-value="'00:00'"><br/>
+			<span ng-if="systems.config[0].wholeHouse[0].otmr[0].includes(':')">Until <ng-timepicker ng-model="systems.config[0].wholeHouse[0].otmr[0]" step="15" init-time="{{ mkTime(systems.config[0].wholeHouse[0].otmr[0]) }}"></ng-timepicker></span>
+			</span>
+		</div>
+		<div class="panel-body">
+			<table class="table table-striped table-bordered">
+				<thead>
+					<tr>
+						<th></th>
+						<th>Dehum. Overcool</th>
+						<th>Cooling Humidity</th>
+						<th ng-if="systems.config[0].cfgvent[0]">Cooling Ventilator</th>
+						<th ng-if="systems.config[0].cfgvent[0]">Cooling Vent. Speed</th>
+						<th ng-if="systems.config[0].cfgvent[0]">Heating Ventilator</th>
+						<th ng-if="systems.config[0].cfgvent[0]">Heating Vent. Speed</th>
+					</tr>
+				</thead>
+				<tbody>
+					<tr>
+						<th>Home</th>
+						<td><input type="checkbox" ng-model="systems.config[0].humidityHome[0].rclgovercool[0]" ng-true-value="'on'" ng-false-value="'off'" /></td>
+						<td>
+							<select name="coolingdehum" ng-model="systems.config[0].humidityHome[0].rclg[0]">
+								<option value="1">46%</option>
```

Comment:
how did you arrive at these values? Poking the thermostat and seeing what changed?

Reply:
Correct, I just went through all the available options on the thermostat and checked what data came back in the JSON.